scripts/deploy_protocol.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import asyncio
+from starknet_py.net.account.account_client import (AccountClient)
+from starknet_py.net.signer.stark_curve_signer import KeyPair
+from starknet_py.contract import Contract
+from starknet_py.net.models import StarknetChainId
+from starkware.crypto.signature.signature import private_to_stark_key
+from starknet_py.net.gateway_client import GatewayClient
+from utils import deployContract
+from pathlib import Path
+
+#Configure Admin AccountClient
+private_key = int(sys.argv[1])
```

Comment:
Could you add an example invoke of this script as a comment?

```
Example usage:

py deploy_protocol.py 0x1248924683bsdf024123abc 0xab717b88a7ba778c86f6f
```

Reply:
Sure,
makes sense.
I like this way because it really prevents people from accidentally pushing a private key to a github repo.
But as long as .env is part of .gitignore I guess it shouldn't be an issue.